docs/enterprise/tripurls.md

```diff
@@ -0,0 +1,64 @@
+
+## Analytics
+
+The `plannedURL` is meant to be used for analytics purposes, to keep track, from the returned trips, which of them the user actually took, if that is possible to determine or guess at a client/app level.
+The `progressURL` goals is to enable apps to report user progress on a specific trip, also for analytics purposes.
+
+
+## Real-time updates
+
+If our platforms has real-time information that can be used to update a computed trip, both `updateURL` and `hookURL` will be present in the response.
+Both support the same goal of updating the trip with real-time information, one by pulling and the other by pushing.
+
+### Pulling for changes
+ 
+The `updateURL` can be used to pull our servers for changes in the trip.
+This url is meant to be used by the apps to get an updated version of the trip.
+This endpoint will return an empty response if there is no change, otherwise, it will return the trip in the requested format.  
+
+As an example, if a service is delayed, the trip will be updated to reflect that change. 
+Also, if the trip involves a booking to a TSP, like **Uber** for example, it will be getting updates of the status of the ride, as soon as those are available from the external API.
+
+The expected flow in this case is the following one:
+
+1. A trip is computed and our platform has sources of information for real-time updates for it, and therefore, the `updateURL` is returned.
+2. The client checks whether the trip has been updated with real-time changes by doing a GET to the `updateURL`.
+3. If the trip has been updated, the new `updateURL` should be used from now on.
+4. After two hours of not getting any update, the trip is discarded from memory.
+
+For more details, check our [docs](/specs/#tag/Trips%2Fpaths%2F~1trip~1update~1%7Bid%7D%2Fget)
+
+### Receiving notifications
+
+The `hookURL` can be used to register a callback or web-hook, and our platform will then inform if any change in that trip occurs by POSTing to the registered url the `tripID` along with the `tripURL` for the client to retrieve the updated trip.
+
+The expected flow in this case is the following one:
+
+1. A trip is computed and our platform has sources of information for real-time updates for it, and therefore, the `hookURL` is returned.
+2. The client that wants to receive notifications about real-time changes does a POST to the `hookURL` ([docs](/specs/#tag/Trips%2Fpaths%2F~1trip~1hook~1%7Bid%7D%2Fpost)) send the webhook as `url` and any required header (starting with `x-`) as body of the request and obtains a 204 response.
+3. At any given time the trip is updated with any real-time change, our platform does a POST to the registered web-hook.
+4. After two hours of not getting any update, the trip is discarded from memory and won't receive any more updates.
```

Comment:
Hmm, but what if I plan a trip for tomorrow morning and want to receive notifications for that? I don't expect any updates overnight, but do expect some in the morning.

I think this would make more sense as: If there are no updates after an hour past the arrival time of the trip.

Reply:
agree, will make that change, and then update the docs